src/pyOpenMS/R-addons/AASequence.R

```diff
@@ -0,0 +1,14 @@
+
+# void getAAFrequencies(Map[String, size_t]) nogil except + # wrap-ignore
+AASequence$$set("public","getAAFrequencies", function(map) {
```

Comment:
What is the reason for the double dollar "$$" referencing. In the reticulate manuals, I only see single dollars "$" for this purpose.